docs/user/openstack/known-issues.md

```diff
@@ -87,6 +87,20 @@ volumeBindingMode: WaitForFirstConsumer
 
 [nova-az-setting]: ../openstack#setting-nova-availability-zones
 
+## Mounting Manila shares to pods
```

Comment:
It bothers me a bit that we're documenting this as a known issue: it is not an issue but instruction on how to connect your pods to the manila network.
I think we should split this into its own "Deploying with Manila" file.

Reply:
Agree, and thanks for getting this documented!